IOEventsWebhook/handler.ts

```diff
@@ -0,0 +1,86 @@
+import * as t from "io-ts";
+import * as express from "express";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "@pagopa/io-functions-commons/dist/src/utils/request_middleware";
+import { RequiredParamMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/required_param";
+import {
+  IResponseSuccessAccepted,
+  ResponseSuccessAccepted
+} from "@pagopa/ts-commons/lib/responses";
+
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import { flow, pipe } from "fp-ts/lib/function";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Context } from "@azure/functions";
+import { ContextMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { RequiredBodyPayloadMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/required_body_payload";
+
+type ExpectedEvents = t.TypeOf<typeof ExpectedEvents>;
+const ExpectedEvents = t.literal("service-subscribed");
+
+type ExpectedEventsWithPayload = t.TypeOf<typeof ExpectedEventsWithPayload>;
+const ExpectedEventsWithPayload = t.interface({
+  name: ExpectedEvents,
+  payload: t.interface({
+    hashedFiscalCode: NonEmptyString
+  })
+});
```

Comment:
Ok, I got cofused by `ExpectedEvents` definition

Reply:
>It's like "here we have a union of types, of which we have just 1 for the moment"

I knew it, just I don't like the approach.

>About this, that's the idea. But I keep them separated because I wanted to discriminate the case we're ignoring an event because it's unexpected and because it has a malformed payload.

This is the reason why, Validation error cannot help with this?